en_us/quests/portfolio_introduction.md

```diff
@@ -0,0 +1,66 @@
+{% include '/version.md' %}
+
+# Portfolio Introduction
+
+## Quest objectives
+
+- Introduce Puppet the company
+- Give an overview of the Puppet portfolio of products
+- Describe Puppet Bolt
+
+## Getting started
+
+Now that you've been introduced to the Quest Guide for the Learning VM, this
+lesson introduces Puppet the company and its portfolio of products.
+
+Puppet was founded in 2005 by Luke Kanies and has offered its first commercial
+product, Puppet Enterprise (PE), since 2011.
+
+Puppet is a global company with headquarters in Portland and worldwide offices
+located in London, Belfast, Sydney, Tokyo and elsewhere. The company has over
+1000 enterprise customers, including 75 of the Fortune 100. Puppet has a robust
+user community that has built many thousands of contributed modules comprising
+over 7.5M lines of code. Additionally, tens of thousands of organizations use
+open-source Puppet to manage their server infrastructure.
+
+Over time, the company has expanded its offerings to the following portfolio
+of enterprise products:
+
+- Puppet Discovery: Identify your IT resources in real time and act on them
+- Puppet Bolt: Perform ad hoc system configuration tasks on your infrastructure
+- Puppet Enterprise: Build model-driven configuration for your IT infrastructure
+- Puppet Pipelines: Simplify your application delivery and deployment process
+- Puppet Insights: Visually measure and optimize your DevOps investment
+
+Puppet Bolt and Puppet Enterprise will be the focus of the Learning VM. We
+invite you to visit [https://puppet.com/products](https://puppet.com/products)
+for additional information of the entire suite of Puppet products.
+
+## Puppet Bolt
+
+Bolt is an open-source remote task runner that automates the manual work that you do
+to maintain your infrastructure. Use Bolt to automate tasks that you need to
+perform on your infrastructure on an ad hoc basis, such as troubleshooting,
+deploying an applications, stopping and starting services, and upgrading a
+database schema. Bolt connects directly to remote nodes with SSH or WinRM, so
+you are not required to install any agent software.
+
+In the Learning VM, Bolt will be installed and used to install the Puppet agent
```

Comment:
This is a great paragraph! I would suggest you consider adding part of this to the first Bolt section paragraph and then dive into the tech talk. I think it answers the question: why am I learning about only this tool in the first quest? But, that really is only a suggestion.